charts/publisher/templates/common/external-secret.yaml

```diff
@@ -1,15 +1,19 @@
+{{ if .Values.externalSecrets.enabled }}
 apiVersion: external-secrets.io/v1alpha1
 kind: ExternalSecret
 metadata:
   name: {{ .Release.Name }}
   labels:
     {{- include "publisher.labels" . | nindent 4 }}
 spec:
-  refreshInterval: "{{ .Values.common.secrets.externalSecretRefreshInterval }}"
+  refreshInterval: {{ .Values.externalSecrets.refreshInterval }}
   secretStoreRef:
     name: aws-secretsmanager
     kind: ClusterSecretStore
   target:
-    name: {{ .Release.Name }}
+    name: {{ .Release.Name }}-env
+  # TODO: Use individual AWS secrets instead of lumping them all together,
+  # so that we can have automated rotation, proper audit trail, etc.
   dataFrom:
-    - key: "{{ .Values.common.secrets.externalSecretRef }}"
+    - key: govuk/publisher
```

Comment:
Maybe minor, but should we not retain this as an explicit Helm value with this value as default, so that it's not tied to a hardcoded key name, to allow multiple deployments with different secrets values in the same AWS account?

Reply:
I completely agree we should be passing these in — I'm only hardcoding this temporarily because I don't want to bake the temporary everything-in-one-AWS-secret approach into the user interface of the chart. Once we've split the secrets up, I expect we'll want to pass a block of YAML in for the dataFrom field, or something like that. I've added a TODO to call that out specifically so I don't forget.

Reply:
(yeah I know literally passing in YAML for dataFrom won't work — probably end up being a list or a map that we expand to a bunch of `data` fields or something like that, possibly even passing the whole spec, though that might be a bit much)